lib/cli/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@alfresco/adf-cli",
   "description": "Alfresco ADF cli and utils",
-  "version": "7.0.0-alpha.6",
+  "version": "7.0.0-alpha.7-0",
   "author": "Hyland Software, Inc. and its affiliates",
```

Comment:
In the same file js-api version should be updated to `"@alfresco/js-api": ">=8.0.0-alpha.7-0",`